graphene/contrib/django/filter/filterset.py

```diff
@@ -0,0 +1,115 @@
+import six
+from django.conf import settings
+from django.db import models
+from django.utils.text import capfirst
+from django_filters import Filter, MultipleChoiceFilter
+from django_filters.filterset import FilterSetMetaclass, FilterSet
+from graphql_relay.node.node import from_global_id
+
+from graphene.contrib.django.forms import GlobalIDFormField, GlobalIDMultipleChoiceField
+
+
+class GlobalIDFilter(Filter):
+    field_class = GlobalIDFormField
+
+    def filter(self, qs, value):
+        gid = from_global_id(value)
+        return super(GlobalIDFilter, self).filter(qs, gid.id)
+
+
+class GlobalIDMultipleChoiceFilter(MultipleChoiceFilter):
+    field_class = GlobalIDMultipleChoiceField
+
+    def filter(self, qs, value):
+        gids = [from_global_id(v).id for v in value]
+        return super(GlobalIDMultipleChoiceFilter, self).filter(qs, gids)
+
+
+ORDER_BY_FIELD = getattr(settings, 'GRAPHENE_ORDER_BY_FIELD', 'order')
```

Comment:
I'm more inclined to use `order_by` as I automatically think of it as a String (field to order), as opposed as using `order` that I think more as a Boolean (if we want to order the results: `True` or `False`).